diff --git a/dist/locale.js b/dist/locale.js
index 9d60215871f1cf1480eef3f4dfdbb32cc8511a1a..442945cff7a744643c0d78c513d297b535d1602c 100644
--- a/dist/locale.js
+++ b/dist/locale.js
@@ -10,6 +10,7 @@ import { GitHubDownloader, INSTALL_DIR, webdl } from "./pkgman.js";
 import { getAsBooleanFromENV } from "./utils.js";
 import { LeakWarning } from "./warnings.js";
 const currentDir = import.meta.dirname ?? path.dirname(fileURLToPath(import.meta.url));
+const TERRITORY_INFO_PATH = process.env.CAMOUFOX_TERRITORY_INFO_PATH ?? path.join(currentDir, "data-files", "territoryInfo.xml");
 export const ALLOW_GEOIP = true;
 class Locale {
     language;
@@ -204,7 +205,7 @@ export async function getGeolocation(ip) {
     return new Geolocation(locale, location.longitude, location.latitude, location.time_zone);
 }
 async function getUnicodeInfo() {
-    const data = await fs.promises.readFile(path.join(currentDir, "data-files", "territoryInfo.xml"));
+    const data = await fs.promises.readFile(TERRITORY_INFO_PATH);
     const parser = new xml2js.Parser();
     return parser.parseStringPromise(data);
 }
diff --git a/dist/webgl/sample.js b/dist/webgl/sample.js
index 3ac1706218d928998cfe7fdf9d9b27481518003b..88a4d04867aa9aa5eed8f3c18ffeea6ef7e9e3bf 100644
--- a/dist/webgl/sample.js
+++ b/dist/webgl/sample.js
@@ -22,7 +22,7 @@ async function openDatabase(pathName) {
 }
 // Get database path relative to this file
 const currentDir = import.meta.dirname ?? path.dirname(fileURLToPath(import.meta.url));
-const DB_PATH = path.join(currentDir, "..", "data-files", "webgl_data.db");
+const DB_PATH = process.env.CAMOUFOX_WEBGL_DATA_PATH ?? path.join(currentDir, "..", "data-files", "webgl_data.db");
 export async function sampleWebGL(os, vendor, renderer) {
     if (!OS_ARCH_MATRIX[os]) {
         throw new Error(`Invalid OS: ${os}. Must be one of: win, mac, lin`);
